data/ships.txt

```diff
@@ -1480,6 +1480,46 @@ ship "Gunboat"
 
 
 
+ship "Harvester"
+	sprite "ship/mining drone"
+	attributes
+		category "Drone"
+		"cost" 100000
+		"shields" 200
+		"hull" 500
+		"mass" 22
+		"drag" .6
+		"heat dissipation" .92
+		"cargo space" 15
+		"outfit space" 56
+		"weapon capacity" 6
+		"engine capacity" 28
```

Comment:
I know some tweaking of the engine output was needed to make the drone nimble enough to be an effective miner, even with the dedicated turn/thrust engines. The only other route was to change the drag or the mass, which are already pretty limited due to it being a human tech drone.